great-idea-website/index.html

```diff
@@ -14,71 +15,79 @@
   <![endif]-->
 </head>
 
-  <body>
-    <header id="home"> <nav>
```

Comment:
a container div with the class of container and a width: { 50% }, margin: { 0, auto } will push everything towards the middle and center it